src/Schema/Schema.php

```diff
@@ -624,34 +651,50 @@ protected function findColumns(TableSchema $table): bool
 ORDER BY
     a.attnum;
 SQL;
+
         $columns = $this->getDb()->createCommand($sql)->queryAll();
+
         if (empty($columns)) {
             return false;
         }
+
         foreach ($columns as $column) {
-            if ($this->getDb()->getSlavePdo()->getAttribute(\PDO::ATTR_CASE) === \PDO::CASE_UPPER) {
-                $column = \array_change_key_case($column, CASE_LOWER);
+            if ($this->getDb()->getSlavePdo()->getAttribute(PDO::ATTR_CASE) === PDO::CASE_UPPER) {
+                $column = array_change_key_case($column, CASE_LOWER);
             }
+
             $column = $this->loadColumnSchema($column);
             $table->columns($column->getName(), $column);
+
             if ($column->isPrimaryKey()) {
                 $table->primaryKey($column->getName());
+
                 if ($table->getSequenceName() === null) {
                     $table->sequenceName($column->getSequenceName());
                 }
+
                 $column->defaultValue(null);
             } elseif ($column->getDefaultValue()) {
-                if ($column->getType() === 'timestamp' && $column->getDefaultValue() === 'now()') {
+                if (
+                    in_array($column->getType(), [self::TYPE_TIMESTAMP, self::TYPE_DATE, self::TYPE_TIME], true) &&
+                    in_array(
+                        $column->getDefaultValue(),
+                        ['now()', 'CURRENT_TIMESTAMP', 'CURRENT_DATE', 'CURRENT_TIME'],
+                        true
+                    )
+                ) {
                     $column->defaultValue(new Expression($column->getDefaultValue()));
                 } elseif ($column->getType() === 'boolean') {
                     $column->defaultValue(($column->getDefaultValue() === 'true'));
-                } elseif (\preg_match("/^B'(.*?)'::/", $column->getDefaultValue(), $matches)) {
-                    $column->defaultValue(\bindec($matches[1]));
-                } elseif (\strncasecmp($column->getDbType(), 'bit', 3) === 0 || \strncasecmp($column->getDbType(), 'varbit', 6) === 0) {
-                    $column->defaultValue(\bindec(\trim($column->getDefaultValue(), 'B\'')));
-                } elseif (\preg_match("/^'(.*?)'::/", $column->getDefaultValue(), $matches)) {
+                } elseif (preg_match("/^B'(.*?)'::/", $column->getDefaultValue(), $matches)) {
+                    $column->defaultValue(bindec($matches[1]));
+                } elseif (
+                    preg_match("#^'(\d+)'::\"bit\"$#", $column->getDefaultValue(), $matches)
```

Comment:
I'd change separators to `/` to match the rest of the code in this file.

```suggestion
                    preg_match("/^'(\d+)'::\"bit\"$/", $column->getDefaultValue(), $matches)
```

Reply:
done.